featurebyte/sql/databricks_unity/F_COUNT_DICT_ENTROPY.sql

```diff
@@ -3,11 +3,15 @@ CREATE OR REPLACE FUNCTION F_COUNT_DICT_ENTROPY(counts MAP<STRING, INT>)
   LANGUAGE PYTHON
 AS
 $$
+    import numpy as np
     from scipy.stats import entropy
     if counts is None:
         return None
     if not counts:
         return 0.0
-    return entropy([abs(v) for v in counts.values()])
+    value = entropy([abs(v) for v in counts.values()])
+    if np.isnan(value):
+        return 0
```

Comment:
Should we return 0 or `None` here?

Reply:
The current behaviour for the Spark and Snowflake UDFs are to return 0 when the input is not empty but the counts sum to 0.